tests/unit/api/bucketPutPolicy.js

```diff
@@ -78,7 +78,7 @@ describe('putBucketPolicy API', () => {
         });
     });
 
-    it('should return error if policy contains conditions', done => {
+    it.skip('should return error if policy contains conditions', done => {
```

Comment:
why are we skipping this test?

Reply:
It's not needed: we now handle policy conditions, so we should not send back error any more. I want confirmation to delete it, all good?

Reply:
We should delete if not needed or re-purpose :)